esp/src/src/Utility.ts

```diff
@@ -107,6 +107,14 @@ export function convertedSize(intsize: number): string {
     }
 }
 
+export function valueCleanUp(intsize): string {
+    if (intsize === null || intsize === undefined || intsize === -1) {
```

Comment:
Should the test for "-1" be here now?

Reply:
I have removed the "-1" from here and have checked pages to make sure it is not needed, where it is needed I have added a check for it.